File: 1014 - consumo/consumo.java
import java.util.Scanner;
import java.io.IOException;
/**
 *
 * 
 *  */
public class Main {
    
    private static float calcularconsumo( int x, float y)
    {
        return ( (float) x / y );
    }
    
    public static void main(String[] args ) throws IOException
    {
        Scanner input = new Scanner( System.in );
        int x;
        float y;
        x = Integer.parseInt(input.nextLine());
        y = Float.parseFloat(input.nextLine());        
        System.out.printf("%.3f km/l\n", calcularconsumo(x,y)  );
    }
    
}

File: 1014 - consumo/consumo.js
//var input = require('fs').readFileSync('stdin', 'utf8');
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

function converter( x, y )
{
    return ( x / y );
}

var x = parseInt( lines.shift() );
var y = parseFloat( lines.shift() );
console.log( (converter(x,y)).toFixed(3) +" km/l");